packs/statuspage_io/CONTRIBUTORS.md

```diff
@@ -0,0 +1,3 @@
+# CONTRIBUTORS
+
+Jon Middleton <jon.middleton@pulsant.com>
```

Comment:
Just a heads up - in new version we added new `contributions` attribute to pack.yaml which is array so now more than one person can be list (most packs have more than one author / contributor these days) :)